moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -810,6 +807,8 @@ void ServoCalcs::suddenHalt(trajectory_msgs::JointTrajectory& joint_trajectory)
   joint_trajectory.points.clear();
   joint_trajectory.points.emplace_back();
   trajectory_msgs::JointTrajectoryPoint& point = joint_trajectory.points.front();
+  point.time_from_start.fromNSec(1);
```

Comment:
This seems a little dangerous. Wouldn't we want to set it to 0, to ensure the joint_trajectory_controller begins stopping as soon as it receives the message?  (per the same wiki section you mentioned, http://wiki.ros.org/joint_trajectory_controller#Trajectory_replacement)

Reply:
Oh, never mind. That's a different message field (header.stamp). I think this is fine.

Reply:
Please add a comment about why you did this.

Reply:
If `time_from_start` is zero and the trajectory start time is also zero, then it's all zeros and that leads to warnings about points being 0 seconds in the past (and like I said in the PR description, it's a bit weird). This is a way to work around that problem.